tools/sz_repo_cli/lib/src/commands/src/deploy_android_command.dart

```diff
@@ -217,22 +224,18 @@ class DeployAndroidCommand extends Command {
     }
   }
 
-  Future<void> _uploadToGooglePlay({
+  Future<void> _uploadToGooglePlay(
+    ProcessRunner processRunner, {
     required String track,
     required String rollout,
   }) async {
-    await runProcess(
-      'fastlane',
-      ['deploy'],
-      workingDirectory: '${_repo.sharezoneFlutterApp.location.path}/android',
-      environment: {
+    await processRunner.run(
+      ['fastlane', 'deploy'],
+      workingDirectory: Directory(
+          path.join(_repo.sharezoneFlutterApp.location.path, '/android')),
+      addedEnvironment: {
         'TRACK': track,
         'ROLLOUT': rollout,
-        // Sets the number of retries for uploading the app bundle to Google
-        // Play. This is needed because sometimes the upload fails for unknown
-        // reasons.
-        //
-        // See: https://github.com/fastlane/fastlane/issues/21507#issuecomment-1723116829
```

Comment:
Wasn't the comment helpful? :(

Reply:
Ah that was on accident, good catch! Its helpful :)